src/main/java/org/myrobotlab/service/config/InMoov2Config.java

```diff
@@ -538,8 +538,7 @@ public Plan getDefault(Plan plan, String name) {
     listeners.add(new Listener("publishStopAnimation", getPeerName("neoPixel")));
     // listeners.add(new Listener("publishProcessMessage",
     // getPeerName("python"), "onPythonMessage"));
-    listeners.add(new Listener("publishProcessMessage", getPeerName("python"), "onPythonMessage"));
-    
+    listeners.add(new Listener("publishProcessMessage", getPeerName("python"), "onPythonMessage"));    
```

Comment:
should the callback method be   onProcessMessage  ?  to follow our normal convention ?  we can refactor this to make it consistent at a later time.